helix-term/src/keymap.rs

```diff
@@ -303,6 +303,14 @@ impl Keymaps {
         self.sticky.as_ref()
     }
 
+    pub fn key_exists(&self, mode: Mode, key: KeyEvent) -> bool {
```

Comment:
To fit with standard data structures like hashmap let's call this `contains_key`